run/CESM/HEMCO_Config.rc

```diff
@@ -3168,250 +3536,250 @@ VerboseOnCores:              root       # Accepted values: root all
-* GMI_LOSS_RCHO       $ROOT/GMI/v2015-02/gmi.clim.RCHO.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   RCHO     - 1  1
-* GMI_PROD_RCHO       $ROOT/GMI/v2015-02/gmi.clim.RCHO.geos5.2x25.nc           prod    2005/1-12/1/0 C xyz v/v/s RCHO     - 1  1
-* GMI_LOSS_RCO3       $ROOT/GMI/v2015-02/gmi.clim.RCO3.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   RCO3     - 1  1
-* GMI_PROD_RCO3       $ROOT/GMI/v2015-02/gmi.clim.RCO3.geos5.2x25.nc           prod    2005/1-12/1/0 C xyz v/v/s RCO3     - 1  1
-* GMI_LOSS_RCOOH      $ROOT/GMI/v2015-02/gmi.clim.RCOOH.geos5.2x25.nc          loss    2005/1-12/1/0 C xyz s-1   RCOOH    - 1  1
-* GMI_PROD_RCOOH      $ROOT/GMI/v2015-02/gmi.clim.RCOOH.geos5.2x25.nc          prod    2005/1-12/1/0 C xyz v/v/s RCOOH    - 1  1
-* GMI_LOSS_RIO1       $ROOT/GMI/v2015-02/gmi.clim.RIO1.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   RIO1     - 1  1
-* GMI_PROD_RIO1       $ROOT/GMI/v2015-02/gmi.clim.RIO1.geos5.2x25.nc           prod    2005/1-12/1/0 C xyz v/v/s RIO1     - 1  1
-* GMI_LOSS_RIPA       $ROOT/GMI/v2015-02/gmi.clim.RIPA.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   RIPA     - 1  1
-* GMI_PROD_RIPA       $ROOT/GMI/v2015-02/gmi.clim.RIPA.geos5.2x25.nc           prod    2005/1-12/1/0 C xyz v/v/s RIPA     - 1  1
-* GMI_LOSS_RIPB       $ROOT/GMI/v2015-02/gmi.clim.RIPB.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   RIPB     - 1  1
-* GMI_PROD_RIPB       $ROOT/GMI/v2015-02/gmi.clim.RIPB.geos5.2x25.nc           prod    2005/1-12/1/0 C xyz v/v/s RIPB     - 1  1
-* GMI_LOSS_RIPD       $ROOT/GMI/v2015-02/gmi.clim.RIPD.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   RIPD     - 1  1
-* GMI_PROD_RIPD       $ROOT/GMI/v2015-02/gmi.clim.RIPD.geos5.2x25.nc           prod    2005/1-12/1/0 C xyz v/v/s RIPD     - 1  1
-* GMI_LOSS_ROH        $ROOT/GMI/v2015-02/gmi.clim.ROH.geos5.2x25.nc            loss    2005/1-12/1/0 C xyz s-1   ROH      - 1  1
-* GMI_PROD_ROH        $ROOT/GMI/v2015-02/gmi.clim.ROH.geos5.2x25.nc            prod    2005/1-12/1/0 C xyz v/v/s ROH      - 1  1
-* GMI_LOSS_RP         $ROOT/GMI/v2015-02/gmi.clim.RP.geos5.2x25.nc             loss    2005/1-12/1/0 C xyz s-1   RP       - 1  1
-* GMI_PROD_RP         $ROOT/GMI/v2015-02/gmi.clim.RP.geos5.2x25.nc             prod    2005/1-12/1/0 C xyz v/v/s RP       - 1  1
-* GMI_LOSS_VRO2       $ROOT/GMI/v2015-02/gmi.clim.VRO2.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   VRO2     - 1  1
-* GMI_PROD_VRO2       $ROOT/GMI/v2015-02/gmi.clim.VRO2.geos5.2x25.nc           prod    2005/1-12/1/0 C xyz v/v/s VRO2     - 1  1
-* GMI_LOSS_VRP        $ROOT/GMI/v2015-02/gmi.clim.VRP.geos5.2x25.nc            loss    2005/1-12/1/0 C xyz s-1   VRP      - 1  1
-* GMI_PROD_VRP        $ROOT/GMI/v2015-02/gmi.clim.VRP.geos5.2x25.nc            prod    2005/1-12/1/0 C xyz v/v/s VRP      - 1  1
+* GMI_LOSS_A3O2       $ROOT/v2015-02/gmi.clim.A3O2.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   A3O2     - 1  1
```

Comment:
@lizziel: Should this section be $ROOT/GMI/v2015-02/... ?

Reply:
Yes, good catch! Those are off by default so my tests did not catch that. I just pushed a fix.